docs/content/latest/explore/change-data-capture/_index.md

```diff
@@ -4,6 +4,7 @@ headerTitle: Change data capture (CDC)
 linkTitle: Change data capture (CDC)
 description: CDC or Change data capture is a process to capture changes made to data in the database.
 image: /images/section_icons/index/develop.png
+beta: /latest/faq/general/#what-is-the-definition-of-the-beta-feature-tag
```

Comment:
Doesn't show on this page; does the tag work on _index pages?

Reply:
The logic was missing from the `list.html` layout. Fixed in 227ff92 just now. 👍